### DISCUSSION TITLE
Developers pls help: white filemenu showed up, can't get rid of it

### DISCUSSION THREAD

**Discussion by ghost** [Open-Shell/Open-Shell-Menu#591] [Q&A]
Here are the screenshots of the bug: https://imgur.com/a/lpu0NPr
First I installed OldNewExplorer, it worked fine. Then I installed Open-Shell with the hope to further customize my file explorer. I'm on windows 10 20h2.
Then,
This ugly file-menu-bar-thing showed up when I installed Open-Shell. Then I went through the settings but there wasn't any setting to close file-menu-bar whatever that is 😄 

I'm stuck with it. I severely want it gone because I was originally trying to simplify the file explorer, not bloat it with another menubar. (In addition to win7 one came with OldNewExplorer). 

I tried every standard thing. I looked at open-shell settings, reinstalled it with file explorer unchecked, uninstalled it again, tried to overwrite the shell with OldNewExplorer, tried using SFC (system file checker) of windows, tried repairing windows, tried every standard thing to do but it is still there. I don't want to clean format windows, I did that too recently 😢 

Can you please help me make it go away, or at least make it dark so it doesn't look so ugly?

Btw. I had a bad experience with it but still I appreciate your work, this happened to me maybe because of me installing OldNewExplorer shell prior to it or whatever but nevertheless you guys are doing a great job, keep it up 👍

**Comment by ghost** [comment]
No one, really?

**Comment by ge0rdi** [comment]
Have you tried to uninstall `Open-Shell`?
Or at least `Classic Explorer` component?

**Comment by NebelNidas** [comment]
Try my fix suggested here: https://github.com/Open-Shell/Open-Shell-Menu/issues/351#issuecomment-640111987
